Description: Fix cleanup in tests.
 Since SQLite 3.15, VACUUM either needs no arguments, or the name of an
 attached database.
 . 
 From https://sqlite.org/lang_vacuum.html :
 Compatibility Warning: The ability to vacuum attached databases was added
 in version 3.15.0 (2016-10-14). Prior to that, a schema-name added to the
 VACUUM statement would be silently ignored and the "main" schema would be
 vacuumed.
 .
 That means that 'VACUUM ANALYZE' never did what it was probably supposed to
 do.
 .
 Maybe 'VACUUM; ANALYZE' works as well. -- And besides this change in the
 testsuite, the documentation for cleanup() in the POD would need to be
 updated as well.
 . 
Origin: vendor
Bug-Debian: https://bugs.debian.org/841545
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-10-21

--- a/t/10_cleanup.t
+++ b/t/10_cleanup.t
@@ -29,7 +29,7 @@
 use strict;
 use ORLite {
 	file    => '$file',
-	cleanup => 'VACUUM ANALYZE',
+	cleanup => 'VACUUM',
 };
 
 1;
--- a/t/11_cleanup.t
+++ b/t/11_cleanup.t
@@ -29,7 +29,7 @@
 use strict;
 use ORLite {
 	file    => '$file',
-	cleanup => 'VACUUM ANALYZE',
+	cleanup => 'VACUUM',
 };
 
 1;
